api/registry/api/v1.py

```diff
@@ -731,12 +733,12 @@ def stamp_display(request) -> List[StampDisplayResponse]:
     # auth=ApiKey(),
     auth=None,
```

Comment:
What is the reason to have no auth here?
Just as a convenience to be able to use that form the staking app as well, without caring about api key?